pnet_packet/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "pnet_packet"
+version = "0.19.0"
+authors = ["Robert Clipsham <robert@octarineparrot.com>"]
+license = "MIT/Apache-2.0"
+homepage = "https://github.com/libpnet/libpnet"
+repository = "https://github.com/libpnet/libpnet"
+description = "Cross-platform, binary packet parsing and manipulation"
+keywords = ["networking", "packet", "parsing", "raw"]
+categories = ["network-programming", "parser-implementations"]
+build = "build.rs"
+
+[dependencies]
+pnet_base = { path = "../pnet_base" }
+pnet_macros_support = { path = "../pnet_macros_support", version = ">=0.1" }
+
+[build-dependencies]
+glob = "0.2"
+syntex = { version = "0.42" }
```

Comment:
Can simply be `syntex = "0.42"` when only version is specified.

Reply:
Sure, I'll fix this.